src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -347,6 +347,23 @@ public Response deleteCluster(@PathParam("cluster_name") String clusterName)
           .entity("cluster \"" + clusterName + "\" not found")
           .build();
     }
+    if (force.orElse(Boolean.FALSE)) {
+      int attempts = 0;
+      do {
+        context.storage.getRepairSchedulesForCluster(clusterName)
+            .forEach((schedule) -> context.storage.deleteRepairSchedule(schedule.getId()));
+
+        context.storage.getRepairRunIdsForCluster(clusterName)
+            .forEach((uuid) -> context.storage.deleteRepairRun(uuid));
+
+        try {
+          Thread.sleep(100);
```

Comment:
Are we deleting at `LOCAL_ONE` in the context of Cassandra, and is that why you're adding a sleep here?

Couldn't we move to a more consistent CL in that case and avoid adding this block?